purchase_order_analytic_search/__openerp__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2015-17 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Purchase Order Analytic Search",
+    "version": "9.0.1.0",
```

Comment:
you missed the last zero. The zeros at the right are always important! :euro: :euro: :euro: :stuck_out_tongue_closed_eyes: